types/run.sh

```diff
@@ -87,12 +87,12 @@ need-typechecking() {
 
 
 typecheck-more-oil() {
-  local log=_tmp/typecheck-more-oil.txt
-
-  set +o errexit
-  cat $MORE_OIL_MANIFEST | xargs -- $0 typecheck $MYPY_FLAGS > $log
-
-  assert-one-error $log
+  # The --follow-imports=silent option allows adding type annotations
+  # in smaller steps without worrying about triggering a bunch of
+  # errors from imports.  In the end, we may want to remove it, since
+  # everything will be annotated anyway.  (that would require
+  # re-adding assert-one-error and its associated cruft, though).
+  cat $MORE_OIL_MANIFEST | xargs -- $0 typecheck --follow-imports=silent $MYPY_FLAGS
```

Comment:
@andychu I decided to just take out the call to assert-one-error here, because it's still used by `typecheck-all` in `types/osh-parse.sh`.  We could also use `--follow-imports=silent` there too, though, and that would allow dropping the function entirely.

I love deleting code, but on a project I'm not so familiar with, I think the parable of "Chesterton's Fence" usually applies :)

Reply:
Yeah that makes sense, it looks like it is still used, but I'm sure we can get rid of it eventually.